src/App.js

```diff
@@ -11,19 +11,17 @@ import 'bootstrap/dist/css/bootstrap.min.css';
 
 function App() {
   const dispatch = useDispatch();
-  const { rocketsReducer: store } = useSelector((state) => state);
-
+  const { rocketsReducer: a, missionReducer: b } = useSelector((state) => state);
```

Comment:
- [ ]  I think it would be better if you give the `rocketReducer` and the `missionReducer` significant names like `rocketStore and missionStore` instead of `a` and  `b`. This will increase the readability of our code.